### PR TITLE
Warning mesheryctl start

### PR DESCRIPTION
**Pull request by Jougan-0**
**Notes for Reviewers**

This PR fixes #

- [x] Add warn command and if docker use give an error command
- [x] docs update for --port-forward command add a note that it is intended to work with docker only
- [x] Note addition to kubernetes getting started page
- [x] Note addition to Guides file

@leecalcote Might need to re-position [this](https://github.com/meshery/meshery/blob/cc9f65488e305b96252e8683caa298f88667e0f7/mesheryctl/internal/cli/root/system/dashboard.go#L74)( placement of include) in dashboard.go because can be seen when using --help command couldn't figure out where to keep it . Currently under Example I am unable to understand where to place them so that it can be detected other than example and long description. The placement in docs gives error.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit fcfa157da3567c2a7289272b914caf1546f184cb at: https://6649471156a6513f8608f6d0--meshery-docs-preview.netlify.app

**Comment by Jougan-0** [comment]
For mesheryctl docs update to be shown do we need to pass something else I wonder?

**Comment by Jougan-0** [comment]
https://github.com/meshery/meshery/actions/runs/9143648489/job/25140595734?pr=10997#step:7:317

this error because the old pr of json returning panic for error
https://github.com/meshery/meshkit/pull/496

needs to merge .

File: mesheryctl/internal/cli/root/system/dashboard.go [hunk 1]
```diff
@@ -105,10 +106,9 @@ mesheryctl system dashboard --skip-browser
 			return nil
 		}
 		log.Debug("Fetching Meshery-UI endpoint")
-
 		switch currCtx.GetPlatform() {
 		case "docker":
-			break
+			log.Warn("The current flag is intended for use exclusively when Meshery is deployed on a Kubernetes platform.")
```
Comment (MUzairS15) [inline mesheryctl/internal/cli/root/system/dashboard.go:111]: Only if `port-forward` flag is used, issue this warning.

**Review by MUzairS15 — CHANGES_REQUESTED** [review]

**Comment by leecalcote** [comment]
merge conflict.

**Comment by l5io** [comment]
🚀 Preview for commit fde18f22be524efb239001b4437b7a5f6cc5d3d2 at: https://664f46b64da29f0099664e48--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 364998e55875a7a648905f45a098f9f896b9ef85 at: https://664f4dd7a662f50b3fd9e8da--meshery-docs-preview.netlify.app

**Comment by leecalcote** [comment]
merge conflict

**Comment by l5io** [comment]
🚀 Preview for commit ca91df044c73f76577d36bfca58a5df2a302c4df at: https://6651fa8839c2b05cd4abbcd6--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 04c91d22eeabd7e0b17568c5dc08fef14064fce5 at: https://6651fc2739c2b05e9eabbede--meshery-docs-preview.netlify.app

File: docs/_includes/mesheryctl/system-dashboard.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+
+Note: The `--port-forward` flag is intended for use exclusively when Meshery is deployed on a "Kubernetes" platform.
```
Comment (leecalcote) [inline docs/_includes/mesheryctl/system-dashboard.md:2]:
```suggestion
Note: The `--port-forward` flag is intended for use exclusively when Meshery is deployed in a Kubernetes cluster (`platform: kubernetes`).
```

**Review by leecalcote — APPROVED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit 703d19984d46a0ec5cf12456da9786bc9705e393 at: https://667d63f6caa5b7d91ff0dcb1--meshery-docs-preview.netlify.app

**Review by leecalcote — APPROVED** [review]

**Pull request merged**